### PR TITLE
[To Be Removed] LogDNA Agent v1.6.4 Pre-Release

### PR DESCRIPTION
**Pull request by smusali**
`LogDNA Agent v1.6.4` is branched off from `LogDNA Agent v1.6.2` not from the `master` since there are untested changes in `master`:

**Done**:
- Update `Dockefile` to use `Debian` package as before
- Restructure the files necessary to build and package things up
- Clean `package.json` to remove unnecessary packages
- Update `CircleCI Config`

**To Be Done**:
- Prepare scripts for each environment

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -21,9 +21,9 @@ module.exports = function(grunt) {
...
-            , fpm_pkg: `fpm -s dir -t osxpkg -n logdna-agent -v ${pkg.version} --license MIT --vendor 'Answerbook, Inc.' --description 'LogDNA Agent for Mac' --url http://logdna.com/ -m '<help@logdna.com>' --after-install ./scripts/mac-after-install --osxpkg-identifier-prefix com.logdna -f ./logdna-agent=/usr/local/bin/logdna-agent ./scripts/com.logdna.logdna-agent.plist=/Library/LaunchDaemons/com.logdna.logdna-agent.plist`
+            , fpm_rpm: `fpm -s dir -t rpm -n logdna-agent -v ${pkg.version} --license MIT --vendor 'Answerbook, Inc.' --description 'LogDNA Agent for Linux' --url http://logdna.com/ -m '<help@logdna.com>' --before-remove ./build/before-remove --after-upgrade ./build/after-upgrade -f ./logdna-agent=/usr/bin/logdna-agent ./build/init-script=/etc/init.d/logdna-agent ./build/logrotate=/etc/logrotate.d/logdna-agent`
```
Comment (mindjiver) [inline Gruntfile.js:24]: Should these say Answerbook Inc?

Comment (smusali) [reply Gruntfile.js:24]: @mindjiver, gonna remove those

**Comment by smusali** [comment]
@mindjiver @gregswift changes to the codebase are done - trying out on Ubuntu

**Comment by smusali** [comment]
This can be closed - another PR will cover this

**Pull request closed without merging**